### DISCUSSION TITLE
Will the Mesh X Maximum Inset be fixed in the next update?

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#918] [Ideas]
I don't know if this was brought up before, but the `Mesh Inset` doesn't exactly save after restart. I think it will save only if the value is greater for `Min`, and less than for `Max`, but the main concern is `Mesh X Maximum` in the _Mesh Inset_ menu. Having a probe automatically sets this value to (**X Bed Size** - **probe.offset.x**). for example 230 - 45. if 230 is max X and 45 is probe offset. so the mesh Inset becomes 185 in order so it can probe the whole bed, basically that is the max point it is able to probe.

however this effects the actual Mesh, and squishes the mesh so it's not able to contain the whole bed. because if G29 P3 is used to probe for the bed Mesh, it automatically fills in the points (for instance the last row it could not otherwise because of the limitation of the bed size and probe offset) with great accuracy.

so it's fine if the probe doesn't reach all the points of the bed, specifically where probe X offset.

is the fix going to be in the next release? where the values can be saved after restart? or whatever math which makes up those values is done away with?

**Comment by mriscoc** [comment]
Currently, the mesh area is the area taken into account for leveling, points outside of the mesh area are extrapolated from the points on the mesh edge. The maximum mesh area is formed from the probing reachable area.

**Reply by classicrocker883** [reply]
yes I see, however that is unnecessary because it limits the usable area of the bed. G29 P3 will fill in any unprobed points automatically very accurately based on all the other points. so that portion of the bed can be usable using a mesh..

that's fine it can account for leveling points outside of the mesh edge, but it doesn't actually save changes to the settings after restart. and that is the issue.

**Reply by JRMSR** [reply]
I just installed latest version and this is still occuring,   maximum X mesh,  is X Max - Probe offset, so mesh does not go all the way to right hand adjusting knobs.  My previous version did,  Why are we liviing with this BUG.  When will it be corrected
